andela/project-kinsan
# #131246955 Authenticated users can view game history

#### What does this PR do?

Adds feature that allows authenticated users to view their game history.
#### Description of Task to be completed?
- A user should be able to click a button which directs to the history page
- A user should see their previous games played on the history page
#### How should this be manually tested?

After login, the user would see the `history` link as part of the top menu on the main page, on the link being clicked the user should be redirected to a history page that displays the users history.
#### What are the relevant pivotal tracker stories?
#131246955 Users should be able to view their personal game history
#### Screenshots (if appropriate)

![project-kinsan-1](https://cloud.githubusercontent.com/assets/20815290/19084639/4b31e904-8a5f-11e6-9aa5-7b15a97d6786.png)
![project-kinsan-2](https://cloud.githubusercontent.com/assets/20815290/19084640/4b330546-8a5f-11e6-93b2-10afe138b1ed.png)
